csi.proto

```diff
@@ -376,6 +382,15 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // Plugin specific mutable parameters passed in as opaque key-value
+  // pairs. This field is OPTIONAL. The Plugin is responsible for 
+  // parsing and validating these parameters. COs will treat these 
+  // as opaque. Plugins MUST treat these as if they take precedence 
+  // over the parameters field.
+  // COs SHALL NOT provide any values in mutable_parameters if the
+  // capability is not enabled.
```

Comment:
You have to mention the capability by name here.

// This field SHALL NOT be specified unless the SP has the
// MODIFY_VOLUME plugin capability.